src/opm/parser/eclipse/share/keywords/000_Eclipse100/W/WFOAM

```diff
@@ -0,0 +1,8 @@
+{
+  "name" : "WFOAM",
+  "sections" : [ "SPECIAL", "SCHEDULE" ],
+  "items" :
+    [{"name" : "WELL"                        , "value_type" : "STRING"},
+     {"name" : "FOAM_CONCENTRATION"          , "value_type" : "UDA" , "dimension" : "FoamDensity"}
```

Comment:
Lo and behold - you are forward leaning; you have a plan here?

Reply:
Not really, beyond trying to not use yesterday's tech here? To me it seems supporting UDA here is no more effort than straight doubles.

Reply:
I guess you are right - so the simple solution here will just be to have `"UDA"` in the json file - as you have, and then when you internalize it in some `EclipseState` object you do:
```
double foam_concentration = record.getItem("FOAM_CONCENTRATION").get<UDAValue>(0).get<double>();
```
which will throw if the input is not a plain double, and then at a next stage we need to pass context to said object to *really* enable UDQ.

Reply:
I see. That means UDQ will not really be enabled for this yet, but the groundwork has been done.

Reply:
That is correct - the parser will handle both `"STRING"` and `0.26` and the parsed data will end up in a `UDAValue` instance, but when fetching data from that instance you must have a plan (i.e. a context available) if you are to handle string/UDA data.